main.tf

```diff
@@ -40,5 +40,52 @@ resource "azurerm_dns_a_record" "wowza" {
   resource_group_name = var.dns_resource_group
   ttl                 = 300
   records             = [var.lb_IPaddress]
-  tags                = local.common_tags
-}
\ No newline at end of file
+
+  tags = local.common_tags
+}
+
+# =================================================================
+# =================    automation account    ======================
+# =================================================================
+resource "azurerm_automation_account" "vm-start-stop" {
+  count = var.azdo_pipe_to_change_vm_status == true ? 1 : 0
+
+  name                = "${var.product}-recordings-${var.env}-aa"
+  location            = var.location
+  resource_group_name = "${var.product}-recordings-${var.env}-rg"
+  sku_name            = var.automation_account_sku_name
+
+  identity {
+    type         = "UserAssigned"
+    identity_ids = [module.vm_automation[0].cvp_aa_mi_id]
+  }
+
+  tags = local.common_tags
+}
+
+#  vm shutdown/start runbook module
+module "vm_automation" {
+  count = var.azdo_pipe_to_change_vm_status == true ? 1 : 0
+
+  source                        = "github.com/hmcts/cnp-module-automation-runbook-start-stop-vm"
+  automation_account_name       = azurerm_automation_account.vm-start-stop[0].name
+  location                      = var.location
+  env                           = var.env
+  resource_group_id             = module.wowza.wowza_rg_id
```

Comment:
if this moves then can use azurerm_resource_group.rg.id and if this moves then can use azurerm_resource_group.rg.location

Reply:
good point, will update to remove dependency.

Reply:
updated to azurerm_resource_group.rg.id and azurerm_resource_group.rg.location